### ISSUE TITLE
[Bug]: Latest release triggers microsoft defender ASR and blocks provider updates.

### ISSUE THREAD

**Issue by DLangenberg** [hashicorp/terraform-provider-aws#40433]
### Terraform Core Version

1.9.7

### AWS Provider Version

5.79.0

### Affected Resource(s)

hashicorp\aws\5.79.0\windows_amd64\terraform-provider-aws_v5.79.0_x5.exe is being blocked by MDE ASR rules, causing inconsistent modules updates 
![image](https://github.com/user-attachments/assets/f0e8f6cc-691c-442c-86e1-be199b49187c)

### Expected Behavior

no ASR block

### Actual Behavior

ASR block

### Terraform Configuration Files

terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 5.17"
    }
    datadog = {
      source  = "datadog/datadog"
      version = "~> 3.27"
    }
    tfe = {
      source  = "hashicorp/tfe"
      version = "~> 0.51"
    }
    gitlab = {
      source  = "gitlabhq/gitlab"
      version = "~> 17.2"
    }
  }
  required_version = ">= 1.3.0"
}

### Steps to Reproduce

terraform init --upgrade 

### Would you like to implement a fix?

None

**Comment by justinretzolk** [comment]
@DLangenberg 👋 Thank you for taking the time to raise this! There've been a couple of releases since you initially raised this. Are you still experiencing the same behavior?

**Comment by justinretzolk** [comment]
Since we haven't heard back, I'm going to close this issue. If you continue to experience unexpected behavior, please feel free to open a new issue with the requested information, and we'll be happy to take another look.
